src/psnawp_api/models/client.py

```diff
@@ -368,6 +369,32 @@ def trophy_groups_summary(
                 np_communication_id=np_communication_id,
             ).user_trophy_groups_summary_with_metadata(account_id="me", platform=platform)
 
+    def title_stats(self, limit: Optional[int] = 100) -> Iterator[TitleStats]:
+        """Retrieve a list of title with their stats and basic meta-data
+
+        :param limit: Limit of titles returned, will default to 100.
+        :type limit: Optional[int]
+
+        .. warning::
+
+            Only returns data for PS4 games and above. Don't put a very high limit as the data returned includes a lot of extra meta data which makes the
+            payload large.
+
+        :returns: List of Titles with their play times
+        :rtype: Iterator[TitleStats]
+
+            .. literalinclude:: examples/client/get_title_stats.json
+                :language: json
+
+
+        .. code-block:: Python
+
+            client = psnawp.me()
+            print(client.title_stats())
+
+        """
+        return TitleStats.from_endpoint(request_builder=self._request_builder, account_id=self.account_id, limit=limit)

```

Comment:
I think you misunderstood. Change the line to 

```py
return TitleStats.from_endpoint(request_builder=self._request_builder, account_id="me", limit=limit)
```

Reply:
Tested it out with `me`, seems to be working fine. Done. 👍🏼